packages/ra-ui-materialui/src/input/DateInput.spec.js

```diff
@@ -32,31 +32,23 @@ describe('<DateInput />', () => {
 
     describe('error message', () => {
         it('should not be displayed if field is pristine', () => {
-            const { container } = render(
```

Comment:
you've removed a useful test

Reply:
I actually kept it but added an error because you can't test it isn't here if it is not defined right ? I removed the should not be displayed if field has been touched but is valid test though as I don't see its point